examples/test-guestbook-spec-change.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: helmrelease-spec-test
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: HelmRelease
+metadata:
+  name: guestbook
+  namespace: helmrelease-spec-test
+repo:
+  chartName: nginx-chart
+  source:
+    github:
+      branch: gh-pages
+      chartPath: nginx-chart
+      urls:
+        - https://github.com/pezhang/helm-release-API-test.git
```

Comment:
I don't think we should point at personal repo like this.

Could you not upload the charts you need to this repo first then reference this repo?